code/game/jobs/job/blackshield.dm

```diff
@@ -127,7 +127,6 @@
 	spawn_positions = 2
 	supervisors = "the Blackshield Commander"
 	difficulty = "Hard."
-	noob_name = "Corpsman Cadet"
 	alt_titles = list("Corpsman Cadet", "Combat Medic")
```

Comment:
```suggestion
	alt_titles = list("Combat Medic")
```

Cadet is a general role (I don't want to have one random trainee alt title after converting the other to a proper role)